src/components/v5/shared/AvatarUser/AvatarUser.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+
+import Avatar from '~v5/shared/Avatar';
+import Link from '~v5/shared/Link';
+import { AvatarUserProps } from './types';
+
+const displayName = 'v5.AvatarUser';
+
+const AvatarUser: FC<AvatarUserProps> = ({
+  walletAddress,
+  avatar,
+  isContributorsList,
```

Comment:
What does this mean? Props shouldn't be related to any logic. They should reflect what you want the component to look like, etc. Right now you're linking this component to a specific place where it's used